### PR TITLE
Created PH for Australia

### PR DESCRIPTION
**Pull request by yourock17**
Created public holiday data for Australia.
yaml code for use in opening_hours.js/holidays

### CODE REVIEW

File: holidays/au.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    'Australia Day': [1, 26]
+    'Good Friday': [easter, -2]
+    'Easter Saturday': [easter, -1]
+    'Easter Monday': [easter, 1]
+    'ANZAC Day': [4, 25]
+    'Labour Day': [firstMayMonday, 0]
+    'Queens Birthday': [firstOctoberMonday, 7]
+    'Christmas Day': [12, 25]
+    'Boxing Day': [12, 26]
+
```
Comment (ypid) [inline holidays/au.yaml]: Note that many well maintained projects have automated tests which are run against each PR. Refer to https://travis-ci.org/opening-hours/opening_hours.js/jobs/279563412#L645. Trailing whitespace is evil :) I fixed it for ya :)

**Comment by ypid** [comment]
Thanks for the update. Here is the updated list:

```
nt: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-04-14 Good Friday
2017-04-15 Easter Saturday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-05-01 May Day
2017-06-12 Queens Birthday
2017-08-07 Picnic Day
2017-12-25 Christmas Day
2017-12-26 Boxing Day
wa: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-03-06 Labour Day
2017-04-14 Good Friday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-06-05 Western Australia Day
2017-12-25 Christmas Day
2017-12-26 Boxing Day
act: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-03-13 Canberra Day
2017-04-14 Good Friday
2017-04-15 Easter Saturday
2017-04-16 Easter Sunday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-05-29 Reconciliation Day
2017-06-12 Queens Birthday
2017-09-25 Family and Community Day
2017-10-02 Labour Day
2017-12-25 Christmas Day
2017-12-26 Boxing Day
vic: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-03-13 Labour Day
2017-04-14 Good Friday
2017-04-15 Easter Saturday
2017-04-16 Easter Sunday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-06-12 Queens Birthday
2017-09-29 AFL Grand Final
2017-11-07 Melbourne Cup
2017-12-25 Christmas Day
2017-12-26 Boxing Day
nsw: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-04-14 Good Friday
2017-04-15 Easter Saturday
2017-04-16 Easter Sunday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-06-12 Queens Birthday
2017-10-02 Labour Day
2017-12-25 Christmas Day
2017-12-26 Boxing Day
qld: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-04-14 Good Friday
2017-04-15 Easter Saturday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-05-01 Labour Day
2017-10-02 Queens Birthday
2017-12-25 Christmas Day
2017-12-26 Boxing Day
tas: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-03-13 Eight Hours Day
2017-04-14 Good Friday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-06-12 Queens Birthday
2017-12-25 Christmas Day
2017-12-26 Boxing Day
sa: 
2017-01-01 New Years Day
2017-01-26 Australia Day
2017-03-13 Adelaide Cup
2017-04-14 Good Friday
2017-04-15 Easter Saturday
2017-04-17 Easter Monday
2017-04-25 ANZAC Day
2017-06-12 Queens Birthday
2017-10-02 Labour Day
2017-12-25 Christmas Day
2017-12-26 Boxing Day
```

Looks good?

**Comment by ypid** [comment]
This PR is almost ready to be merged. Just one last check from you would be nice :)

**Comment by yourock17** [comment]
The dates are correct. They were corrected in the update. Thank you :)

**Comment by ypid** [comment]
Merged. Thanks, you are welcome :)

**Pull request merged**